src/org.freedesktop.fwupd.xml

```diff
@@ -1075,6 +1075,33 @@
       </arg>
     </method>
 
+    <!--***********************************************************-->
+    <method name='Search'>
+      <doc:doc>
+        <doc:description>
+          <doc:para>
+            Return search data.
+          </doc:para>
+        </doc:description>
+      </doc:doc>
+      <arg type='s' name='token' direction='in'>
+        <doc:doc>
+          <doc:summary>
+            <doc:para>
+              A search term.
+            </doc:para>
+          </doc:summary>
+        </doc:doc>
+      </arg>
+      <arg type='aa{sv}' name='results' direction='out'>
+        <doc:doc>
+          <doc:summary>
+            <doc:para>Releases that match the search term</doc:para>
+          </doc:summary>
+        </doc:doc>
+      </arg>
+    </method>
```

Comment:
Where is this documentation published? @hughsie 
I'd like to add some examples, I was trying to figure out what search terms I can use and what fields it searches.
Right now I have to look at the source code, which is not very convenient for users.